Frontend/CO.CDP.OrganisationApp/Authorization/CustomScopeHandler.cs

```diff
@@ -23,7 +23,7 @@ protected override async Task HandleRequirementAsync(AuthorizationHandlerContext
                     var organisationUserScopes = await _userInfo.GetOrganisationUserScopes();
 
                     // Admin role can do anything within this organisation
-                    if (organisationUserScopes.Contains(OrganisationPersonScopes.Admin))
+                    if (organisationUserScopes.Contains(OrganisationPersonScopes.Admin) && requirement.Scope != PersonScopes.SupportAdmin)
```

Comment:
Doesn't seems right, if the person is admin for organisation he should be able to perform anything for that organisation, regardless of person scope

Reply:
Yeah, you're right. This does not cover that scenario.

Reply:
Wouldn't the next pass of requirement.Scope equals ADMIN, EDITOR or VIEWER satisfy this condition?

Reply:
This isn't testing the person's scope though, it's checking that the _requirement_ that we're testing for is not support admin.

If the person is admin, but the thing we're looking for is support admin, don't let them through.

Reply:
@andymantell  what if support admin has a "test organisation" to do checks?
Also if that is not possible scenario, than also we are sure that Person with 'SupportAdmin' scope will never be and Admin on an Organisation hence the above condition will never met?

Reply:
That doesn't matter - because we're testing the _requirement_, not the presence of support admin on their user. This line is saying, if the _requirement_ is asking for support admin, but the current person is admin, don't allow them through.

Reply:
Wait I see what you're saying I think

Reply:
Yeah. I think the line of code is just confusing because its checking for the presence of an organisation scope on the left and then checking that there isn't a SupportAdmin person scope requirement on the right.

I'll refactor to make the logic's intention clearer